implementation/test/DepositFraudTest.js

```diff
@@ -341,6 +344,102 @@ describe("DepositFraud", async function() {
     })
   })
 
+  describe("startSignerFraudLiquidation", async () => {
+    let signerBond
+    before(async () => {
+      signerBond = 10000000
+      await ecdsaKeepStub.send(signerBond, {from: accounts[1]})
+    })
+
+    beforeEach(async () => {
+      await createSnapshot()
+    })
+
+    afterEach(async () => {
+      await restoreSnapshot()
+    })
+
+    it("executes and emits StartedLiquidation event", async () => {
+      const block = await web3.eth.getBlock("latest")
+
+      await testDeposit.startSignerFraudLiquidation({from: owner})
+
+      const events = await tbtcSystemStub.getPastEvents("StartedLiquidation", {
+        fromBlock: block.number,
+        toBlock: "latest",
+      })
```

Comment:
In digging at truffle-contract for tbtc.js, I found that you can directly pull events from the transaction receipt:

https://github.com/keep-network/tbtc.js/blob/5e1f35a459e95ff0d8f33286f37ea0a006173d34/src/EthereumHelpers.js#L1-L25

May be worth considering this for our tests, rather than `getPastEvents`. No need to do that in this PR though, just a thought/flag. That may also allow us to use e.g. `buidler`, which would improve our dev experience more as well. It would also remove a block wait from the start of these tests. Also not sure if that applies to OpenZeppelin's setup as well.

Reply:
This will be easy for events that don't go through the Deposit logging. Can use openzeppelin `expectEvent`.
As you probably know from tbtc.js events that are executed outside the context of the called function don't get included in the logs and are rather encoded in rawLogs. It's less direct to retrieve them from there. Still very possible and probably better.